### ISSUE TITLE
Question about some variables

### ISSUE THREAD

**Issue by alliecuenco** [ahmetozlu/vehicle_counting_tensorflow#40]
What does "bottom" and "bottom_position_of_detected_vehicle" variable mean? I get so confused. Does bottom refer to the bottom coordinates of the current object then the bottom_position_of_detected_vehicle is the previous bottom position of the previous vehicle?

**Comment by ChampionTej05** [comment]
Yes, you got it right. We are saving the previous detected location to improve the accuracy while counting the objects. It is so that, if the previously object was detected near the ROI and now it is just down ROI by 5 or 10 (those values 205 and 210), then we say that object crossed the line and count it as 1.

**Comment by ZIKO94ZIKO** [comment]
why he chose this values (205 and 210) exactly and how he found this value 44 in (speed_prediction.py line 56)  > scale_real_length` = pixel_length * 44

**Comment by ZIKO94ZIKO** [comment]
@ChampionTej05  any update
